numba_dppy/tests/test_dpnp_functions.py

```diff
@@ -267,6 +264,732 @@ def f(a):
             self.assertTrue(np.array_equal(expected, got))
 
 
+@unittest.skipUnless(ensure_dpnp(), "test only when dpNP is available")
+class Testdpnp_random_functions(unittest.TestCase):
+    sizes = [None, 9, (2, 5), (3, 2, 4)]
+
+    def test_random_sample(self):
+        from numba.tests.support import captured_stdout
+
+        @njit
+        def f(size):
+            c = np.random.random_sample(size)
+            return c
+
+        sizes = [9, (2, 5), (3, 2, 4)]
+        set_dpnp_debug(1)
```

Comment:
Can we transform pair `set_dpnp_debug(1)` and `set_dpnp_debug(None)` into with context like `with dpnp_debug(): ...`?

Reply:
We can.